plexpy/graphs.py

```diff
@@ -165,17 +161,26 @@ def get_total_plays_per_day(self, time_range='30', y_axis='plays', user_id=None,
                   'series': series_output}
         return output
 
+    def make_user_cond(self, user_id):
+        """
+        Expects user_id to be a comma-separated list of ints.
+        """
+        user_cond = ''
+        if session.get_session_user_id() and user_id and user_id != str(session.get_session_user_id()):
+            user_cond = 'AND session_history.user_id = %s ' % session.get_session_user_id()
+        elif user_id:
+            user_ids = [id.strip() for id in user_id.split(',')]
```

Comment:
There is a helper function for this.
```suggestion
            user_ids = helpers.split_strip(user_id)
```